### ISSUE TITLE
libRNFS missing when archiving

### ISSUE THREAD

**Issue by benorudolf** [itinance/react-native-fs#591]
Hi, 

I've setup react-native-fs via react-native link command and proceeded to link it manually in XCode as well - everything is in place as per doc. 
A debug build on simulator builds and works fine, but archiving fails with the error:
`clang: error: no such file or directory: '/Users/benorudolf/Library/Developer/Xcode/DerivedData/VertragsAtlas-deqolswfglqofdeznxegsrtdwtin/Build/Intermediates.noindex/ArchiveIntermediates/VertragsAtlas/BuildProductsPath/Release-iphoneos/libRNFS.a'`

I've checked the DerivedData folders and the `libRNFS.a` is actually missing from `Release-iphoneos` - it seems to be in the `RNFS` folder inside the `Release-iphoneos`. 
I tried to create the alias manually but the error persisted. 

I went through deleting the derived data several times, product cleans as well, but the error remains. 

I'm guessing it's a linking issue, but can't seem to find where I failed as everything seems as per doc.

**Comment by benorudolf** [comment]
Attaching some info on my setup: 

**Podfile**  
![image](https://user-images.githubusercontent.com/955836/49009340-9ef09c00-f170-11e8-89b0-77633fbd6c58.png)

**Xcode**
![image](https://user-images.githubusercontent.com/955836/49009372-baf43d80-f170-11e8-96f7-b182197d3031.png)

**Comment by Stekmo** [comment]
i'm experiencing the same issue running on react-native @0.54.3 and react-native-fs @2.11.17. 

@benorudolf did you solve this?

**Comment by benorudolf** [comment]
@Stekmo  
Yes I did manage to resolve it by unlinking the React lib from the Pods Targets in Xcode. It was one of the **many** solutions available around the web and it was the one that worked. 

**Please mind, that I have no idea if this actually resolves this concrete issue nor if it's even related, but it helped me get a working build.** 

Open your xcworkspace file, browse the Project files, click on Pods and on the right side under targets delete the React package.
![image](https://user-images.githubusercontent.com/955836/49575356-c3076680-f942-11e8-9a1d-d77c0948a243.png)
